src/Controller/BatchProcessingController.py

```diff
@@ -320,6 +325,38 @@ def batch_dvh2csv_handler(self, interrupt_flag,
         self.batch_summary[patient]['dvh2csv'] = reason
         progress_callback.emit(("Completed DVH2CSV", 100))
 
+    def batch_pyrad2csv_handler(self, interrupt_flag,
+                                progress_callback, patient):
+        """
+        Handles creating, starting, and processing the results of batch
+        Pyrad2CSV.
+        :param interrupt_flag: A threading.Event() object that tells the
+                               function to stop loading.
+        :param progress_callback: A signal that receives the current
+                                  progress of the loading.
+        :param patient: The patient to perform this process on.
+        """
+        # Get current files
+        cur_patient_files = self.get_patient_files(patient)
```

Comment:
Same problem as previous PR here. Static method should not be called by an instance

Reply:
These have been fixed. Thank you for picking these up! Must have missed them before I created the PR.